target/linux/mediatek/dts/mt7981a-glinet-gl-xe3000.dts

```diff
@@ -0,0 +1,8 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include "mt7981a-glinet-gl-x3000-xe3000-common.dtsi"
+
+/ {
+	model = "GL.iNet GL-XE3000";
+	compatible = "glinet,gl-xe3000", "mediatek,mt7981";
```

Comment:
So the only difference is the name?!

In the case, please use a single image and add `DEVICE_ALT_*` fields in `target/linux/mediatek/image/filogic.mk`.

Reply:
There are no differences in the device tree between the two devices, except for the name, yes. However, the XE3000 device has a battery, while the X3000 does not have one. 

As far as I can see, merging both devices in a single image using `DEVICE_ALTX_*` fields would mean having the same identifier in `/proc/device-tree/compatible` and `/proc/device-tree/model`: wouldn't it be an issue for the users trying to identify the devices after flashing the image?

But I'm not familiar with the way such similar devices are handled in OpenWrt, so if you confirm that I should use a single image for both devices, I will do the change.

Reply:
The general rule is: If there are no software differences justifying the need for another image (and hence continuous need to spend resources on build and download servers), then we shouldn't create another image.
If there is a way to interact with the PMIC/charger e.g. via I2C on the XE3000 device (ie. to know the current battery level, charging/discharging current, ...), then of course that I2C device should be defined in DT and would be such a software difference.

Reply:
Ack. I don't see any USB involved there, but never mind. In the long run it would be better to implement this as a serdev driver in kernel, exposing the usual hwmon interface instead of having a custom tool in userspace, but for now this is fine.

Reply:
Please could you mark the request change as addressed so that this can be merged?